oak/proto/storage.proto

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright 2019 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak;
+
+message ReadRequest {
+  bytes data_store_id = 1;
+  bytes name = 2;
+}
+
+message ReadResponse {
+  bytes value = 1;
+}
+
+message WriteRequest {
```

Comment:
It seems like we would need a `transaction_id` field here as well, otherwise how are we going to associate r/w operations to transactions?

Reply:
Done.